samples/remote_attestation/common/attestation.cpp

```diff
@@ -47,14 +47,14 @@ bool Attestation::generate_remote_evidence(
     // Generate evidence based on the format selected by the attester.
     result = oe_get_evidence(
```

Comment:
Can we showcase the custom claim serialization helper functions here?

Reply:
Sure

Reply:
The sample here use the public key pem as the custom claim for the evidence. The key was later used to establish secure communications between two enclaves. It might not be a good fit to place the key pem into a name-value structure.

Reply:
Sure, that's a good idea.